styles/footer.css

```diff
@@ -0,0 +1,82 @@
+.footer-container {
+    justify-content: flex-start;
+    background-color: #6dc5dd;
+    box-sizing: border-box;
+    padding: 56px 0 1px 0;
+    border-radius: 0 0 5px 5px;
+}
+
+.footer {
+    justify-content: space-between;
+    color: white;
+    min-height: 260px;
+    height: 100%;
+}
+
+.footer__list {
+    width: 100%;
+    margin-bottom: 34px;
+
+    display: grid;
+    grid-gap: 30px;
+    grid-template-columns:  repeat(6, max-content);
+    justify-content: space-between;  
+}
+
+.footer-article__title {
+    margin-bottom: 9px; 
+    letter-spacing: 1px;   
+}
+
+.footer-article__prh {
+    margin-bottom: 18px; 
+    line-height: 27px;
+    letter-spacing: 0.6px;
+    font-size: 14px;   
+}
+
+.footer__underline {
+    display: flex;
```

Comment:
Yey! This is the case when using flexbox is completely justified. It would be great to use appropriate tools to solve appropriate problems.